apps/api/v1/pages/api/credential-sync/_post.ts

```diff
@@ -108,7 +108,7 @@ async function handler(req: NextApiRequest) {
   // ^ Workaround for the select in `create` not working
 
   if (createCalendarResources) {
-    const calendar = await getCalendar(credential);
+    const calendar = await getCalendar({ ...credential, delegatedTo: null });
```

Comment:
`getCalendar` now requires CredentialForCalendarService which has an explicit value for `delegatedTo`